amundsen_application/static/css/_variables-default.scss

```diff
@@ -26,7 +26,13 @@ $gray-lightest: lighten($gray-base, 95%) !default; // #f2f2f2
 
 // Scaffolding
 $body-bg:    #fff !default;
-$text-color: $gray-dark !default;
+// TODO - replace $text-color with the new $text-light, medium, dark
+
+$text-dark: $gray-dark !default;
+$text-medium: $gray !default;
+$text-light: $gray-lighter !default;
+$text-color: $text-dark !default;
```

Comment:
Instead of picking one of the 7 gray colors for our text fields, we should explicitly define our text colors. I like having `$text-dark`, `$text-medium`, and `$text-light`.

`$text-color` is used only by Bootstrap to define the `color` on the `body`, but we could also replace `$text-dark` and use `$text-color`, `$text-medium`, and `$text-light` instead.